frontend/src/pages/facemash.js

```diff
@@ -80,33 +78,41 @@ class FaceMash extends Component<Props> {
       }
       this.getNewMatch()
     } catch (err) {
-      console.error(('Error': err))
+      console.error('Error', err)
     }
   }
 
   render() {
     if (this.props.error) {
-      return <div>Bad Fetch. Try again</div>
+      return <Error>Bad Fetch. Try again</Error>
     }
     const { candidates } = this.props
     return candidates && candidates.length == 2 ? (
       <div>
         <Container>
+          <div className="mt-5" style={{ color: '#7f8e9e' }}>
+            <h4>Decide impartially on who you think would be more successful at Hack4Impact.</h4>
+            <p>
+              Although we need as many matches made as possible, please still take the time to look
+              think critically when making your choice. Each decision you make will affect each
+              candidate's ability to move on to the interview rounds.
```

Comment:
Can we change the quote to `&#39;`. Otherwise the syntax highlighting breaks.

Reply:
yep